### DISCUSSION TITLE
even though i am in the technical preview copilot insiders is asking me to join the waitlist.

### DISCUSSION THREAD

**Discussion by TarunavBA** [community/community#7044] [Copilot Conversations] [answered]
even though I am on the technical preview, copilot insiders is asking me to join the waitlist, So @davecheney can you please re-accept me into the waitlist? The account is https://www.github.com/TarunavBA/

**Comment by davecheney** [comment, marked as answer]
I have confirmed you are enabled for the Copilot technical preview.

Your token has not made any requests to our servers. This is most likely because of a configuration issues on your machine, or a network issue between your machine and our servers.

**Comment by samualtarpenning** [comment]
@davecheney Copilot was working then the icon in the status bar went away after clicking it. I have signed in and out reinstalled the extension etc. Any help would be appreciated. 

vs code version: 1.61.2
copilot version: 1.6.3571

Version: 1.61.2 (user setup)
Commit: 6cba118ac49a1b88332f312a8f67186f7f3c1643
Date: 2021-10-19T14:57:20.575Z
Electron: 13.5.1
Chrome: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Windows_NT x64 10.0.19042

**Reply by davecheney** [reply]
<img width="475" alt="Screen Shot 2021-11-03 at 6 13 26 am" src="https://user-images.githubusercontent.com/7171/139929908-6db0f80b-6f9a-4ce7-b34d-8749a8949832.png">

Clicking the icon disables copilot. You can re-enable it from the extensions menu
